tt_metal/impl/dispatch/command_queue.hpp

```diff
@@ -553,13 +587,19 @@ class CommandQueue {
     bool async_mode() { return this->mode == CommandQueueMode::ASYNC; }
     bool passthrough_mode() { return this->mode == CommandQueueMode::PASSTHROUGH; }
 
-    static CommandQueueMode get_mode() {
-        // Envvar is used for bringup and debug only. Will be removed in the future and should not be relied on in production.
-        int value = parse_env<int>("TT_METAL_CQ_ASYNC_MODE", static_cast<int>(CommandQueueMode::PASSTHROUGH));
-        return static_cast<CommandQueue::CommandQueueMode>(value);
-    }
+    std::atomic<std::size_t> worker_thread_id = -1;
+    std::atomic<std::size_t> main_thread_id = -1;
```

Comment:
let's use parent - worker thread naming scheme, so we don't assume 'main' thread

Reply:
Done